tests/test_amazon_music.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+import json
+from selenium import webdriver
+from pages.header import Header
+from pages.amazon_music import AmazonMusic
+
+with open("config.json", "r") as file:
+    config = json.load(file)
+
+@pytest.fixture
+def driver():
+    driver = webdriver.Chrome()
+    driver.maximize_window()
+    yield driver
+    driver.quit()
+
+def test_amazon_music(driver):
+    driver.get(config.get("BASE_URL"))
+
+    header = Header(driver)
+    header.click_music()
+
+    amazon_music = AmazonMusic(driver)
+
+    print("Page title:", driver.title)
+    print("URL:", driver.current_url)
+
+    current_path = driver.current_url.replace(config.get("BASE_URL").rstrip("/"), "")
```

Comment:
mucha lógica acá, es necesario hacer todo ese trabajo?
en la validación siguiente estas buscando el path "in" el current_path (contains), por ende no deberia molestarle la BASE_URL en esa validación, ademas el strip está puesto por la barra extra en el archivo de config no ?

Reply:
```suggestion
    current_path = driver.current_url
```
asi deberia ser suficiente